LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaToolsTest/LoRaLibTest.cs

```diff
@@ -36,7 +36,7 @@ public void TestJoinAccept()
                 2, 3, 25, 128,

```

Comment:
Please revert the changes to using type instead of var for variable declarations.

Reply:
Done.